R/layouts.R

```diff
@@ -251,3 +251,63 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+#' Flow layout
+#'
+#' Lays out elements in a left-to-right, top-to-bottom arrangement.
+#' The elements on a given row will be top-aligned with each other.
+#'
+#' The width of the elements and spacing between them is configurable.
+#' Lengths can be given as numeric values (interpreted as pixels)
+#' or character values (interpreted as CSS lengths).
+#' With the default settings this layout closely resembles the `flowLayout`
+#' from Shiny.
+#'
+#' @param ...
+#' Unnamed arguments will become child elements of the layout.
```

Comment:
Please keep the parameter and description in the same line

Reply:
Done.